src/GraphQL/Validation/ValidationContext.cs

```diff
@@ -570,7 +570,7 @@ public string IsValidLiteralValue(IGraphType type, IValue valueAst)
                     : $"Expected type '{type.Name}', found {valueAst.StringFrom(Document)}.";
             }
 
-            throw new ArgumentOutOfRangeException(nameof(type), $"Type {type?.Name} is not a valid input graph type.");
+            throw new ArgumentOutOfRangeException(nameof(type), $"Type {type?.Name ?? "<NULL>"} is not a valid input graph type.");
```

Comment:
@Shane32  Previously, there was an strange message with empty type, because `type` was `null`. I added `<NULL>` literal to explicitly state this but now with new checks inside schema validation visitor it should not happen. Of cource one may "turn off" our validation so better safe than sorry.